pepr/robustness/art_wrapper.py

```diff
@@ -0,0 +1,3281 @@
+"""PePR wrapper classes for ART attack classes."""
+
+import logging
+import numpy as np
+import os
+
+from pepr.attack import Attack
+from pepr import report
+import matplotlib.pyplot as plt
+from pylatex import Command, Tabular, MiniPage, NoEscape, Figure
+from pylatex.section import Subsubsection
+from pylatex.utils import italic
```

Comment:
Ich think we do not need `italic` :)

Reply:
Yep, I experimented with it and forgot to remove it, thanks!